prefect_gcp/utilities.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional
+
+from slugify import slugify
+
+
+def _slugify_name(name: str, max_length: int = 30) -> Optional[str]:
```

Comment:
tiny nit, if we're importing this elsewhere, its not really private, but nothing is in python anyways so 🤷🏼

Reply:
true, I did just rip it from https://github.com/PrefectHQ/prefect-kubernetes/blob/main/prefect_kubernetes/utilities.py#L114

Reply:
not an excuse though. I'll fix